src/components/BaseMap/BaseMap.module.scss

```
@@ -47,3 +94,35 @@
 }
+
+// Replace MapLibre's default geolocate icon with the classic Material Icons
+// my_location (24x24 viewBox, evenly-stroked outer circle). Two SVG variants
+// (different fill colors) so the shape stays the same across states — only the
+// colour changes when tracking is active. SVGs are fully URL-encoded so the
+// data URIs don't contain raw <, >, or " characters (keeps SCSS language
+// servers happy when parsing the file).
+$my-location-svg-prefix: 'data:image/svg+xml;utf8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20height%3D%2224%22%20viewBox%3D%220%200%2024%2024%22%20width%3D%2224%22%20fill%3D%22%23';
+$my-location-svg-suffix: '%22%3E%3Cpath%20d%3D%22M12%208c-2.21%200-4%201.79-4%204s1.79%204%204%204%204-1.79%204-4-1.79-4-4-4zm8.94%203A8.994%208.994%200%200%200%2013%203.06V1h-2v2.06A8.994%208.994%200%200%200%203.06%2011H1v2h2.06A8.994%208.994%200%200%200%2011%2020.94V23h2v-2.06A8.994%208.994%200%200%200%2020.94%2013H23v-2h-2.06zM12%2019c-3.87%200-7-3.13-7-7s3.13-7%207-7%207%203.13%207%207-3.13%207-7%207z%22%2F%3E%3C%2Fsvg%3E';
+
+:global(.maplibregl-ctrl button.maplibregl-ctrl-geolocate .maplibregl-ctrl-icon) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)